src/features/firstLogin/PrivacyPolicy/PrivacyPolicyModal.tsx

```diff
@@ -34,7 +35,7 @@ export const PrivacyPolicyModal: FunctionComponent<Props> = ({
     <AppModal
       visible={visible}
       title={t`Respect de ta vie privée`}
-      disableBackdropTap={disableBackdropTap}
+      onBackdropPress={disableBackdropTap ? noop : undefined}
```

Comment:
```suggestion
      onBackdropPress={noop}
```

car `disableBackdropTap` est toujours true (tu peux enlever la prop.